gs.js

```diff
@@ -120,6 +124,36 @@ class GlobalStorage extends lib.StorageProvider {
   ) {
     return this.nextId++;
   }
+
+  // Returns: <number>, get system suffix for given Id
+  getSystemSuffix(id) {
+    return static and(id, this.systemBitmask);
```

Comment:
What type `id` is supposed to be? If this is `number` then it should be
```suggestion
    return id & this.systemBitmask;
```
Or, in case if `id` is `common.Uint64`
```suggestion
    return common.Uint64.and(id, this.systemBitmask);
```